site/search/Search.scss

```diff
@@ -271,16 +271,53 @@
     font-size: 0.9em;
 }
 
+@keyframes chartErrorFadeIn {
+    from {
+        opacity: 0;
+    }
+    to {
+        opacity: 1;
+    }
+}
+
 .search-results__chart-hit-img-container {
     margin-bottom: 8px;
+    aspect-ratio: 850 / 600;
```

Comment:
I'll just note here that [browser support for `aspect-ratio`](https://caniuse.com/mdn-css_properties_aspect-ratio) is currently 94% - it has been implemented in all major browsers in 2021.

If the user's browser doesn't support it, things will still work - the only thing that's happening is that the error tile will not be full-height.
I think this is absolutely acceptable behaviour for a small minority of browser share.